app/src/main/java/com/example/searchmovies/view/MovieFragment.kt

```diff
@@ -31,10 +33,21 @@ class MovieFragment : Fragment(R.layout.fragment_movie) {
 
     private fun handleMovies() {
         viewModel.getSearchedMovies().observe(viewLifecycleOwner) { it ->
-            val movieContent = it.movies
-                .joinToString { it.movieContent }
-            view_content_video.text = movieContent
+            view_content_video.text = it.moviesContent
+            view_content_best_video.text =
+                HtmlCompat.fromHtml(
+                    "<b>${it.theBestMovie.title} </b> <br> " +
+                            " ${it.theBestMovie.releaseDate} <br>" +
+                            " ${it.theBestMovie.voteAverage} <br> <br>" +
+                            " ${it.theBestMovie.overview}"
```

Comment:
Zakładałem, że będziesz chciał to zrobić w kilku TextView - osobny na tytuł, datę wydania, ocenę itp. Takie podejście ułatwia wprowadzanie ewentualnych modyfikacji widoku w przyszłości. 
Jeśli jednak chcesz to zrobić w 1 TextView, to kod HTML powinien być tworzony na etapie mapowania warstwy domenowej na warstwę ViewEntity. ViewEntity powinno przechowywać dane w takiej formie, w jakiej chcesz je wyświetlić. Więc masz 2 opcje :).